src/content/meetup/2306-openreply.md

```diff
@@ -1,5 +1,5 @@
 ---
-date: '2023-06-15T18:30:00+02:00'
+date: 2023-06-15T18:30:00+02:00
 location:
   name: 'Open Reply'
```

Comment:
There is this concept of referencing data: https://docs.astro.build/en/guides/content-collections/#defining-collection-references

I personally think the current implementation (copy / paste) is good enough for the current usecase :)
Just want to mention the reference collections, if someone want to have some fun.